GruntFile.js

```diff
@@ -377,16 +344,14 @@ module.exports = function (grunt) {
 
     grunt.registerTask('default', [
         'jasmine:main:build',
-        'jshint:force',
-        'jscs:force',
-        'if-missing:esri_slurp:dev',
+        'eslint:main',
```

Comment:
Will `:force` work here? It's nice not to have to keep restarting grunt when you have errors to begin with.